Frontend/flight-search-amadeus/package.json

```diff
@@ -13,6 +13,7 @@
     "@popperjs/core": "^2.11.8",
     "@reduxjs/toolkit": "^2.2.7",
     "bootstrap": "^5.3.3",
+    "moment": "^2.30.1",
```

Comment:
Moment is decrypted, please use another library https://momentjs.com/docs/#/-project-status/recommendations/

Reply:
Noted